examples/01-filter/image-representations.py

```diff
@@ -0,0 +1,223 @@
+"""
+.. _image_representations_example:
+
+Image Data Representations
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+This example demonstrates how to use :meth:`~pyvista.ImageDataFilters.points_to_cells`
+and :meth:`~pyvista.ImageDataFilters.cells_to_points` to re-mesh :class:`~pyvista.ImageData`.
+
+These filters can be used to ensure that image data has an appropriate representation
+when generating plots and/or when using either point- or cell-based filters such as
+:meth:`ImageDataFilters.image_threshold <pyvista.ImageDataFilters.image_threshold>` (point-based)
+and :meth:`DataSetFilters.threshold <pyvista.DataSetFilters.threshold>` (cell-based).
+
+"""
+
+################################################################################
+# Representations of 3D Volumes
+# -----------------------------
+# Create image data of a 3D volume with eight points and a discrete scalar data
+# array.
+
+import numpy as np
+
+import pyvista as pv
+
+# sphinx_gallery_thumbnail_number = 3
+
+
+data_array = [8, 7, 6, 5, 4, 3, 2, 1]
+points_volume = pv.ImageData(dimensions=(2, 2, 2))
+points_volume.point_data['Data'] = data_array
+
+################################################################################
+# If we plot the volume, it is represented as a single cell with eight points,
+# and the point data is interpolated to color the cell.
+
+points_volume.plot(show_edges=True)
+
+################################################################################
+# However, in many applications (e.g. 3D medical imaging) the scalar data arrays
+# represent discretized samples at the centers of voxels. As such, it may
+# be more appropriate to represent the data as eight voxel cells instead of
+# eight points. We can use :meth:`~pyvista.ImageDataFilters.points_to_cells` to
+# generate a cell-based representation.
```

Comment:
```suggestion
# However, in many applications (e.g. 3D medical imaging), the scalar data arrays
# represent discretized samples at the centers of voxels. As such, it may
# be more appropriate to represent the data as eight voxel cells instead of
# eight points. We can use :meth:`~pyvista.ImageDataFilters.points_to_cells` to
# generate a cell-based representation.
```